benchmark/python/benchmark_e2e.py

```diff
@@ -196,19 +197,40 @@ def main(args):
         avg_sampling_latency_ms,
         avg_wall_clock_thrpt,
         avg_wall_clock_time,
-    ]]
+    ]
+    return metrics
 
+def main(args):
+    all_csv_metrics = []
+    for model_path in args.input_folders:
+        print("Benchmarking " + model_path)
+        for batch_size in args.batch_sizes:
+            for l, prompt_length in enumerate(args.prompt_lengths):
+                for g, gen_length in enumerate(args.generation_lengths):
+                    if args.max_lengths:
+                        m = l * len(args.generation_lengths) + g
+                        max_length = args.max_lengths[m]
+                    print(f"Args: batch_size = {batch_size}, prompt_length = {prompt_length}, tokens = {gen_length}, max_length = {max_length}")
+                    metrics = run_benchmark(args, model_path, batch_size, prompt_length, gen_length, max_length)
```

Comment:
we should load model only once and run for different batch, prompt and gen_length